tutorial/tutorial-template-guide.md

```diff
@@ -0,0 +1,132 @@
+* By the end of this tutorial, you'll be able to create a private channel in Slack.
+
+If you need help with creating learning objectives, consider using Arizona State University's [Learning Objectives Builder tool](https://teachonline.asu.edu/objectives-builder/).
+
+### About the "Background" section
+
+The purpose of the background section is to provide any necessary context for your users before they start your tutorial. 
+
+If you're writing a tutorial that's focused on learning how a particular feature works, you might describe that feature here. For example, a tutorial around using Microsoft Visio's templates to build flowcharts might start with an explanation of how flowcharts are used or what flowchart shapes represent. 
+
+If you're writing a tutorial where you provide users with a starter project, you could describe the project hierarchy here. For example, a tutorial around building a website with HTML, CSS, and JavaScript might explain how the folders are organized, what code is provided for the user, or what they need to add.
+
+### About the "Before you begin" section
+
+Use this section to list any prerequisites for completing the tutorial. This may include operating systems, languages, package managers, or software.
+
+### About the "Steps" section
+
+The steps section is where you describe what the user needs to do to complete the tutorial. The template organizes steps in the following way:
+
+```
+1. {Write the command here. Start with a verb.}
+
+   {Optional: Explanatory text}
+
+   {Optional: Code sample or screenshot that helps your users complete this step}
+
+   {Optional: Result}
+
+```
+
+The first element is the step itself, like "Enter a name for your new repository." When writing steps, keep these tips in mind:
+
+* Start with an imperative verb. For example, "connect", "set up", or "arrange." Don't use the *-ing* form of the verb because it is harder to translate.
+* Express steps as a complete thought. Instead of saying, "Set up access", you might say, "Set up access to a Cloud Storage bucket."
+
+Next, you may choose to add some additional information about the step. Continuing with the example, you might provide some best practices for creating memorable repository names.
+
+Optionally, you can add a code sample or screenshot after the explanatory text, depending on the type of tutorial you're writing. Screenshots are a great way to show specific parts of the screen you are referring to in a step. 
+
+If you're including code samples in your repository, make sure to add comments to your code to help learners understand what each part of the code does. Additionally, make sure your code samples work, as learners will copy and paste your code to use in their own projects.
+
+The last, and optional, element in a step, is the result. Continuing with the example, you might describe what happens after a user click the "Create repository" button.
+
+Here is an example step:
```

Comment:
**Suggestion (non-blocking):** I like your examples here. Can you also add an example of a code sample with a comment since you mentioned that advice?